toxav/bwcontroller.h

```diff
@@ -5,19 +5,24 @@
 #ifndef C_TOXCORE_TOXAV_BWCONTROLLER_H
 #define C_TOXCORE_TOXAV_BWCONTROLLER_H
 
-#include "../toxcore/Messenger.h"
+#include <stdint.h>
```

Comment:
standard headers always after your own headers.

Reply:
ok so if its everywhere then we can not solve it in this PR. its a different topic then